xdsl/dialects/builtin.py

```diff
@@ -588,6 +588,20 @@ def normalized_value(
 
         return value
 
+    def get_normalized_value(self, value: int) -> int:
```

Comment:
Probably worth either making private or adding tests.

Reply:
it wouldn't work as private, I'll add tests